virttest/test_setup.py

```diff
@@ -645,7 +646,12 @@ def setup(self):
                       self.hugepage_size)
         logging.debug("Number of large memory pages needed for this test: %s",
                       self.target_hugepages)
-        self.set_hugepages()
+        if self.target_nodes:
+            for node in self.target_nodes.split():
+                self.set_node_num_huge_pages(self.target_hugepages, node,
```

Comment:
And, I think we should divide the target_hugepages across nodes rite ? 
we get overall hugepages number using get_target_hugepages or `target_hugepages` param, but here we try to set all the hugepages in each node instead of dividing across nodes.